packages/edit-site/src/components/sidebar-navigation-screen-navigation-menus/leaf-more-menu.js

```diff
@@ -82,9 +75,6 @@ export default function LeafMoreMenu( props ) {
 					{
 						postType: 'page',
 						postId: attributes.id,
-						...( isPreviewingTheme() && {
-							wp_theme_preview: currentlyPreviewingTheme(),
-						} ),
```

Comment:
Preserving `wp_theme_preview` no longer needs to be done explicitly like this. It is now, and should continue to be, abstracted and automated.

Reply:
Does [this place](https://github.com/WordPress/gutenberg/blob/07ea95344c5343d8f8f8c8f3ce71de9a66174df8/packages/edit-site/src/components/routes/link.js#L36) need to remove the param management?

Reply:
Does [this place](https://github.com/WordPress/gutenberg/blob/07ea95344c5343d8f8f8c8f3ce71de9a66174df8/packages/edit-site/src/components/routes/link.js#L36) need to remove the param management?

Reply:
That `useLink` function is also creating a URL link that can be used as an `<a href>` attribute. Therefore the `wp_theme_preview` arg needs to be added explicitly, there is no router magic that would add it automatically. That's why I'm keeping it here.